cwltest/utils.py

```diff
@@ -28,8 +27,9 @@ def create_test_case(self, test):
             category = ", ".join(test['tags'])
         else:
             category = REQUIRED
+        short_name = test.get(u'short_name')
         case = junit_xml.TestCase(
-            doc, elapsed_sec=self.duration, classname=self.classname,
+            doc, elapsed_sec=self.duration, classname=short_name,
```

Comment:
but that will conflict with `--classname`

It seems that the `name` attribute is unused: https://github.com/kyrus/python-junit-xml/blob/master/junit_xml/__init__.py#L326

Reply:
`Name` is the first argument and is initialized by the doc string. I used `file` attribute now.